src/components/editor/EditorAppController.jsx

```diff
@@ -25,24 +25,29 @@ export default class EditorAppController extends BaseComponent {
     }
   }
 
-  isrestarted (){
+  isRestarted (){
     http.get('/isrestarted', (response) => {
       let signal = '';
+      let count = 0;
       response.on('data', (chunk) => {
         signal += chunk;
       });
       response.on('end', () => {
-          if (signal === 'false'){
-              window.alert("Servers restarted successfully");
+          if (signal === 'false') {
+            window.alert("Servers restarted successfully");
             }
-          else if (signal === 'true'){
-              window.alert('Pinging Servers');
-              this.isrestarted();
+          else if (signal === 'true') {
+            while (count < 5) {
```

Comment:
Ikenna, I'm sorry if this sounds a little rough, but are you testing the code?

This loop would run 5 times instantly and set 5 timers that will all execute at pretty much the same time.